Makefile

```diff
@@ -115,6 +115,21 @@ build/host/:
 $(VERSION_FILE):
 	echo $(GIT_VERSION) > $@
 
+.PHONY: snapshots
```

Comment:
I think all of these should go into the cmake file (also because it has dependency tracking), but that's for a different PR.